package.json

```diff
@@ -30,6 +30,7 @@
     "babel-preset-latest": "^6.22.0",
     "babel-preset-react": "^6.22.0",
     "eslint": "^3.16.1",
+    "eslint-config-keystone": "^3.0.0",
```

Comment:
I'm removing it in the PR I'm working on, so just let me know if it is needed in some way but I can't find it being used anywhere.